lib/cookie.c

```diff
@@ -1230,7 +1230,7 @@ struct CookieInfo *Curl_cookie_init(struct Curl_easy *data,
      */
     remove_expired(c);
 
-    if(fromfile && fp)
+    if(fromfile)
```

Comment:
If `fopen()` fails on line 1197, `fromfile` is set but `fp` isn't and shouldn't be `fclose`d. Thus, we need this check.

Reply:
I see, then this is not important.

Reply:
> Thus, we need this check.

no we don't, it's in the if(fp) block